### ISSUE TITLE
Is there a way to use the EfficientNet applications?

### ISSUE THREAD

**Issue by gsteinbu** [rstudio/keras#1075]
In the python version of keras there are applications for EfficientNet (see https://keras.io/api/applications/) Is there a way I can use/call them from R (the keras R packages does not seem to support them yet)? I guess that is similar to whats asked in #848, however, I am not necessarily interested in the pre-trained weights, but the architecture itself. Thanks for any help!

**Comment by dfalbel** [comment]
The applications functions are simple wrapers around the keras api:
See:

```
> keras::application_mobilenet
function(input_shape = NULL, alpha = 1.0, depth_multiplier = 1, dropout = 0.001, 
                                  include_top = TRUE, weights = "imagenet", input_tensor = NULL, pooling = NULL, 
                                  classes = 1000) {
  keras$applications$MobileNet(
    input_shape = normalize_shape(input_shape),
    alpha = alpha,
    depth_multiplier = as.integer(depth_multiplier),
    dropout = dropout,
    include_top = include_top,
    weights = weights,
    input_tensor = input_tensor,
    pooling = pooling,
    classes = as.integer(classes)
  )
}
<environment: namespace:keras>
```

Do you want to make a PR adding efficient net wrapers?

**Comment by gsteinbu** [comment]
Just created respective pull request, thanks @dfalbel. I noticed that the EfficientNet models seem to be available only in TensorFlow >= 2.3.0. Since TensorFlow 2.3.0 hasn't been fully released yet (rc2 has just been released), it might be a good idea to wait a bit with merging my pull request.
